back/db/schema.rb

```diff
@@ -967,6 +950,11 @@
     t.index ["args"], name: "que_jobs_args_gin_idx", opclass: :jsonb_path_ops, using: :gin
     t.index ["data"], name: "que_jobs_data_gin_idx", opclass: :jsonb_path_ops, using: :gin
     t.index ["queue", "priority", "run_at", "id"], name: "que_poll_idx", where: "((finished_at IS NULL) AND (expired_at IS NULL))"
+    t.check_constraint "(char_length(last_error_message) <= 500) AND (char_length(last_error_backtrace) <= 10000)", name: "error_length"
+    # t.check_constraint "(jsonb_typeof(data) = 'object'::text) AND ((NOT (data ? 'tags'::text)) OR ((jsonb_typeof((data -> 'tags'::text)) = 'array'::text) AND (jsonb_array_length((data -> 'tags'::text)) <= 5) AND que_validate_tags((data -> 'tags'::text))))", name: "valid_data"
```

Comment:
Is this comment generated and not manually added? Do you know why it's commented? And why were these changes added at all to `que_jobs` table?

Reply:
It seems it has been happening in master for ages. I've run `docker-compose run web rails db:drop db:create db:migrate` in f3b9309d07 (Oct 26) and these lines appeared.

I assume we can remove or ignore it for now and fix it later.

Reply:
Yes indeed, this issue was already there. The problem is with the Que migration which generates these `check_constraint`s. In order to keep the schema as close as possible to the migrations I kept as much as possible, but commented out the lines that result in DB errors. I tried to briefly look into the issue but couldn't find any similar issues on the internet and couldn't find a solution.